testsuite/forge/src/runner.rs

```diff
@@ -323,6 +329,13 @@ impl ForgeConfig {
             if let Some(storage_gib) = fullnode_resource_override.storage_gib {
                 helm_values["fullnode"]["storage"]["size"] = format!("{}Gi", storage_gib).into();
             }
+
+            if retain_all_logs {
+                helm_values["validator"]["podAnnotations"]["aptos.dev/log-levels"] =
+                    serde_yaml::Value::String("warn,info,error".to_owned());
```

Comment:
How about debug logs? I think that's the most important one for us. Also we usually set `hyper=off` otherwise the logs gets polluted. 

Also, how does this interact with the values.yaml file https://github.com/aptos-labs/aptos-core/blob/d90bdcc9b85f36cd3e5476c4ad12263d03cad1c1/testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml#L5

Reply:
Updated this to include debug logs

This annotation is only used for the Vector filter so the `rust_log` value should still work

Reply:
I see, but from a UX perspective, we now need to set the boolean and update aptos-node-default-values.yaml. Can we automatically set that to `debug,hyper=off` if we set retail_all_logs?

Reply:
Updated to also set `rust_log`